docs/docs/get-started.md

```diff
@@ -3,13 +3,68 @@ title: Getting Started
+This guide walks you through how to use Recce, whether you're exploring for the first time or expanding to Recce Cloud.
 
-Recce can get your schema from all data warehouses, so you don't need to update the profiles.yml. However, if you use duckdb, you need to add a separate schemas in your dbt profile so you can run locally.Your `profiles.yml` might look something like this:
+Use Recce to validate your dbt changes with clarity and confidence. Recce helps you:
 
+- Explore what changed
+- Validate downstream impacts
+- Collaborate through shareable checklists
+
+For a hands-on walkthrough, check out the [Jaffle Shop Tutorial](https://www.notion.so/infuseai/get-started-jaffle-shop.md).
+
+## Start Recce with two commands
+
+Navigate to your dbt project and run:
+```shell
+cd your-dbt-project/  # if you're not already there
+pip install -U recce
+recce server
+```
+
+Recce use dbt [artifacts](https://docs.getdbt.com/reference/artifacts/dbt-artifacts) generated with every invocation. You can find these files in the `target/` folder.
+
+| artifacts     | dbt command                        |
+| ------------- | ---------------------------------- |
+| manifest.json | `dbt docs generate`, `dbt run`, .. |
+| catalog.json  | `dbt docs generate`                |
+
+!!! tip
+
+    The regeneration of the `catalog.json` file is not required after every `dbt run`. it is only required to regenerate this file when models or columns are added or updated.
+
+## Get instant visibility
+
+With just one environment, Recce gives you:
+
+- **Lineage clarity**: Trace changes down to the column level
+
+- **Query insights**: Explore logic and run custom queries
+
+- **Live diffing**: Reload and inspect changes as you iterate
+
+Perfect for early exploration, root cause analysis, and faster debugging before involving others.
+
+## Unlock diff & validation with two environments
+Not all data changes are obviously right or wrong. Comparing metrics before and after is key to confident validation.
+
+When you configure two dbt environments (e.g. prod and dev schemas), Recce lets you:
+
+- **Explore** modified models and downstream impact
+
+- **Validate** changes via diffs or custom queries
+
+- Add validation results to **Checklists** for review and alignment
+
+
+### How to setup two environments
+Setup two separate [environments](https://docs.getdbt.com/docs/environments-in-dbt) that refers to different [schemas](https://docs.getdbt.com/docs/core/connect-data-platform/connection-profiles#understanding-target-schemas), e.g., prod for production and dev for development.
+
+Recce supports schema discovery across all major warehouses without needing to modify your `profiles.yml`.
+
+However, if you're using DuckDB, you’ll need to explicitly define both schemas. For example:
```

Comment:
Hi Karen,
> However, if you use duckdb, you need to add a separate schemas in your dbt profile so you can run locally

I think we need to keep the original explanation -> "it's to make DuckDB able to run locally"

Reply:
updated. I'll merge first in case I don't have time to do that next week. and we can update later